Нужно больше функций

Magnolia42850 opened the pull request:
Подключил скрип и добавил функции

---
:mortar_board: [Нужно больше функций](https://up.htmlacademy.ru/javascript/27/user/2106747/tasks/6)

:boom: https://htmlacademy-javascript.github.io/2106747-kekstagram-27/2/

Magnolia42850 committed:
Add files via upload

Добавлены функции для проверки длинны строки и возвращения случайного числа из диапазона

Magnolia42850 committed:
Add files via upload

подключён скрипт main.js

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+// Функция, возвращающая случайное целое число из переданного диапазона включительно
```
не забываем удалять комментарии

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+// Функция, возвращающая случайное целое число из переданного диапазона включительно
+function getRandomInt(min, max) {
```
Предлагаю сразу писать функции в Arrow function expressions

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+// Функция, возвращающая случайное целое число из переданного диапазона включительно
+function getRandomInt(min, max) {
+    if (min < 0 || max < 0 || min > max) {
```
Где больше двух условий обычно их сохраняют  в переменную предикат 

https://javascopes.com/predicate-programming-logic-7f3-c6d6865a/

например тут можно использовать isArgumentsValid

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+// Функция, возвращающая случайное целое число из переданного диапазона включительно
+function getRandomInt(min, max) {
+    if (min < 0 || max < 0 || min > max) {
+        return NaN;
```
Предлагаю тут выбросить ошибку 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+
+//Функция для проверки максимальной длины строки
+function getVeryfiMaxLength(str, maxLength) {
+    if (str.length <= maxLength) {
```
тут условие не нужно можно просто из функции вернуть str.length <= maxLength это и будет булево значение

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,16 @@
+        return Math.floor(min + Math.random() * (max + 1 - min));
+  };
+
+//Функция для проверки максимальной длины строки
```
Комментарии удаляем

NikolayUsov commented:
Для коммуникации предлагаю писать мне в тг.

Magnolia42850 committed:
Add files via upload

Исправил замечания ( убрал комментарии, сделал функции стрелочными, добавил переменную предикат)

Magnolia42850 commented:
Исправил замечания ( убрал комментарии, сделал функции стрелочными, добавил переменную предикат)

NikolayUsov commented on main.js:
```diff
@@ -0,0 +1,9 @@
+let getRandomInt = (min, max) => {
+    const isArgumentsValid = (min >= 0 && max >= 0 && min < max);
```
скобки необязальны
